src/gt4py/next/iterator/ir_utils/ir_makers.py

```diff
@@ -356,6 +363,18 @@ def lifted_neighbors(offset, it) -> itir.Expr:
     return lift(lambda_("it")(neighbors(offset, "it")))(it)
 
 
+def as_fieldop_neighbors(offset, it) -> itir.Expr:
```

Comment:
We need to also pass the optional domain of `as_fieldop`, at least for unit testing of gtir to SDFG. Anyway, I could also add it in my PR.